coalib/results/Diff.py

```diff
@@ -352,6 +352,7 @@ def delete_line(self, line_nr):
         """
         Mark the given line nr as deleted. The first line is line number 1.
         """
+        line_nr = 0 if (len(self._file) < line_nr) else line_nr
```

Comment:
@RohanVB Well  I agree with you this format is not in use in this file, but this format is actually in use in many places in the codebase of coala.